apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/VideoBlockEditorSettings.tsx

```diff
@@ -74,33 +78,51 @@ export function VideoBlockEditorSettings({
   })
 
   return (
-    <Box sx={{ p: 3, width: '100%' }}>
+    <Box sx={{ px: 6, py: 3, width: '100%' }}>
       <Stack direction="column" spacing={3}>
         <Stack direction="row" justifyContent="space-between">
           <Stack direction="column">
-            <Typography variant="subtitle2">Autoplay</Typography>
-            <Typography variant="caption">
+            <Typography
+              variant="subtitle2"
+              color={disabled ? theme.palette.grey[400] : ''}
```

Comment:
```suggestion
              color={disabled ? theme.palette.action.disabled : ''}
```
Stick to our predefined tokens in `colors.ts` as much as possible, if we don't have anything suitable - that's an indication the design is off or we need to introduce a new token.